CDP4IMEInstaller/ThirdPartyLicense/Moq.txt

```diff
@@ -0,0 +1,34 @@
+﻿name: Moq
```

Comment:
the test related libs do not need to be here as they are not being distributed with the installer, please remove